packages/generate-html/test/generate-html.spec.ts

```diff
@@ -97,8 +144,8 @@ describe('GenerateHTML', function () {
       }, 8000);
 
       it('should render HTML certificate using loaded certificate and HBS template', async () => {
-        const certificate = JSON.parse(readFileSync(certificatePath, 'utf8'));
-        const html = await generateHtml(certificate);
+        const generateHtmlOptions = localOnly ? localOnlyOptions : {};
+        const html = await generateHtml(certificate, generateHtmlOptions);
```

Comment:
Now that we use partials for Handlebars templates, we also need to take care of loading partials even remotely.
To do so, you can use the same logic as in the `parseHbsTemplate` function.